### PR TITLE
MAT-8010: convert editor to npm partial

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-8010](https://jira.cms.gov/browse/MAT-8010)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,10 +78,9 @@
     "@madie/madie-admin": "^0.0.1",
     "@madie/madie-auth": "^0.0.2",
     "@madie/madie-design-system": "^1.2.92",
-    "@madie/madie-editor": "^0.0.2",
     "@madie/madie-layout": "file:",
     "@madie/madie-models": "^1.4.65",
-    "@madie/madie-root": "^0.0.2",
+    "@madie/madie-root": "file:../madie-root/madie-madie-root-0.0.3.tgz",
```
Comment (RohitKandimalla) [inline package.json]: We may want to revert this before merging

**Review by RohitKandimalla — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (mcmcphillips) [reply package.json]: reverted

**Review by RohitKandimalla — APPROVED** [review]
